deployments/data-hub/release-data-hub--stg.yaml

```diff
@@ -100,6 +102,8 @@ spec:
       - name: CIVICRM_EMAIL_DATA_CONFIG_FILE_PATH
         value: /dag_config_files/civicrm-email-report-data-pipeline.config.yaml
       # scheduler interval
+      - name: ELIFE_ARTICLE_XML_PIPELINE_SCHEDULE_INTERVAL
+        value: "*/75 * * * *" # At every 75th minute. It will be changed after initial load.
```

Comment:
does this actually work with values over 60?

Using [crontab.guru](https://crontab.guru/#*/75_*_*_*_*), which may be incorrect, the trigger time doesn't change with any value over 60.